packages/phrases/src/locales/en.ts

```diff
@@ -324,6 +324,7 @@ const translation = {
       branding: {
         title: 'BRANDING',
         primary_color: 'Primary color',
+        dark_primary_color: 'Primary color(Dark)',
```

Comment:
Should be `'Primary color (Dark)'`

<img width="530" alt="image" src="https://user-images.githubusercontent.com/10594507/163102989-d633a950-a68a-4269-bc30-cfab83dab41b.png">

Reply:
updated